Helm/opensearch-dashboards/templates/role.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: {{ template "opensearch-dashboards.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "opensearch-dashboards.standard" . | indent 4 }}
+rules:
+- apiGroups: ['extensions']
+  resources: ['podsecuritypolicies']
+  verbs:     ['use']
```

Comment:
There are too many trailing spaces after `verbs:` here.

Reply:
@smlx that's probably intentional for readability.

Reply:
Yes, @DandyDeveloper is correct.